src/map/options.ts

```diff
@@ -243,6 +243,11 @@ export class MapOptions extends OptionsGroup {
                         scaled = 1.0 / scaled;
                         break;
                     case 'log':
+                        if (scaled <= 0) {
+                            sendWarning(
```

Comment:
Copy/pasting from #121:

This will send multiple warnings if the property contains multiple negative values. I think it would be better to move this check out of the rawSizes.map call; and only warn once.

Reply:
Sounds good. I'm trying to find a good spot for it. Perhaps right before the if-statement that contains the rawSizes.map call? I was thinking that it would warn before `scaleMode` is declared/assigned, but not sure if there is a way to change the scale mode the user has selected by that point anyway.